Weather/Views/TimedForecastCellView.swift

```diff
@@ -12,6 +12,32 @@ class TimedForecastCellView: UICollectionViewCell {
     let timeLabel = UILabel()
     let temperatureLabel = UILabel()
     
+    var presenter: TimedForecastPresenter?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        presenter = TimedForecastPresenterImplementation()
+        
+        timeLabel.text = "2 AM"
+        timeLabel.font = UIFont.boldSystemFont(ofSize: 17)
+        timeLabel.translatesAutoresizingMaskIntoConstraints = false
+        timeLabel.textColor = UIColor.white
+        addSubview(timeLabel)
+        activateTimeLabelConstraints(view: timeLabel)
+        
+        temperatureLabel.text = "2°"
+        temperatureLabel.font = UIFont.boldSystemFont(ofSize: 17)
+        temperatureLabel.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
`translatesAutoresizingMaskIntoConstraints` should be inside `activateConstraints` func.